nifi-nar-bundles/nifi-hadoop-bundle/nifi-hdfs-processors/src/main/java/org/apache/nifi/processors/hadoop/ListHDFS.java

```diff
@@ -462,11 +523,15 @@ private String getPerms(final FsAction action) {
 
     private PathFilter createPathFilter(final ProcessContext context) {
         final Pattern filePattern = Pattern.compile(context.getProperty(FILE_FILTER).getValue());
```

Comment:
Does this need to support expression language?

Reply:
@ottobackwards The FILE_FILTER property does not currently support expression language.  The processor could be updated to enable EL for the property, but that is outside the scope of this PR.

Reply:
fair enough